Makefile

```diff
@@ -139,6 +139,11 @@ android-ports-avd: ##@other Add reverse proxy to Android Device/Simulator
 	adb -e reverse tcp:3449 tcp:3449
 	adb -e reverse tcp:4567 tcp:4567
 
+android-ports-genymotion: ##@other Add reverse proxy to Android Device/Simulator
```

Comment:
is this part of PR's scope?

Reply:
yes, to test it you need to run in on android :)